apis/ops/v1alpha1/constant.go

```diff
@@ -110,6 +115,14 @@ const (
 	RestartPodsFailed    = "RestartPodsFailed"
 )
 
+// Reload
+const (
+	Reload              = "Reload"
+	ReloadPods          = "ReloadPods"
```

Comment:
Why both `Reload` & `ReloadPods` ? 

`ReloadPods` is enough I think.  For whole db, you will use `Reconfigure` or similar word. Not reload